portal/get_patient_info.php

```diff
@@ -279,6 +284,18 @@
     header('Location: ' . $landingpage . '&w');
     exit();
 }
+
+// now that we are authorized, we need to check for the redirect, sanitize it (or eliminate it if we can't), and then redirect
+
+if (isset($_REQUEST['redirect'])) {
```

Comment:
For me it is easier and safer to set supers to a var at top. This way if the super gets modified or somehow corrupted before an eval you know at least you are safe.
By now, is_set is a flag w/o content eval and previously eval'ed so prob is okay.